connectors/google-sheets/element-templates/google-sheets-connector.json

```diff
@@ -0,0 +1,586 @@
+      "description": "Enter the id of the worksheet",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.worksheetId"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "maxLength": 100
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "deleteWorkSheet",
+          "addEmptyColumnOrRow",
+          "createHeaders"
+        ]
+      }
+    },
+    {
+      "label": "Worksheet id",
+      "description": "Enter the id of the worksheet",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.worksheetId"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "deleteColumn"
+        ]
+      }
+    },
+    {
+      "label": "Worksheet name",
+      "description": "Enter name for the worksheet",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.worksheetName"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "maxLength": 100
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createWorksheet",
+          "worksheetData"
+        ]
+      }
+    },
+    {
+      "label": "Worksheet name",
+      "description": "Enter the name of the worksheet",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.worksheetName"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "getRowByIndex",
+          "createRow",
+          "addValues"
+        ]
+      }
+    },
+    {
+      "label": "Dimension",
+      "description": "Choose what to add: column or row",
+      "group": "operationDetails",
+      "type": "Dropdown",
+      "optional": false,
+      "choices": [
+        {
+          "name": "Column",
+          "value": "COLUMNS"
+        },
+        {
+          "name": "Row",
+          "value": "ROWS"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.dimension"
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "addEmptyColumnOrRow"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Row index",
+      "description": "Enter row index",
```

Comment:
can you add an example here

Reply:
<img width="373" alt="image" src="https://github.com/camunda/connectors-bundle/assets/131355515/1c8564ca-9aa7-4a5b-ac8f-1581343ae33c">